backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -313,7 +313,7 @@ public Response unlock(@PathParam("vaultId") UUID vaultId, @QueryParam("evenIfAr
 		}
 
 		var user = userRepo.findById(jwt.getSubject());
-		if (user.getPublicKey() == null) {
+		if (user.getEcdhPublicKey() == null) {
```

Comment:
Correct, users don't necessarily have this field set. However only those users who have an ECDH key can invoke this method, otherwise http status code 449 is expected as per contract.